### PR TITLE
dasharo-security/20F-me-neuter.md: add and update test cases

### PR DESCRIPTION
**Pull request by Pokisiekk**
Signed-off-by: Kamil Pokornicki <kamil.pokornicki@3mdeb.com>

### CODE REVIEW

File: docs/unified-test-documentation/dasharo-security/20F-me-neuter.md [hunk 1]
```diff
@@ -13,11 +13,144 @@
...
+    [Test cases common documentation](#test-cases-common-documentation) section.
+
+**Test steps**
+
+1. Power on the DUT.
+1. While the DUT is booting, hold the `BIOS_SETUP_KEY` to enter the UEFI Setup
+    Menu.
+1. Enter the `Dasharo System Features` menu using the arrow keys and Enter.
+1. Enter the `Dasharo Security Options` submenu.
+1. Verify the `DME HAP disable` field.
```
Comment (macpijan) [inline docs/unified-test-documentation/dasharo-security/20F-me-neuter.md:39]: @sulewskiprzemyslaw @Pokisiekk `DME` ? We mentioned that there will be one option with 3 selectable entries.

The option name: `ME state`
You enter that menu and then you can select from list three options: `Enabled` `Soft-disabled` `HAP-disabled`

Comment (macpijan) [reply docs/unified-test-documentation/dasharo-security/20F-me-neuter.md:39]: @mkopec CC

Comment (mkopec) [reply docs/unified-test-documentation/dasharo-security/20F-me-neuter.md:39]: I've named the options as such - let me know if you want to change them @macpijan : 
image

Comment (mkopec) [reply docs/unified-test-documentation/dasharo-security/20F-me-neuter.md:39]: Moreover the Intel Management Options submenu is located directly in the Dasharo System Features Menu, not inside the Dasharo Security Options menu.

Comment (Pokisiekk) [reply docs/unified-test-documentation/dasharo-security/20F-me-neuter.md]: I have updated the documentation according to the given nomenclature.

**Review by miczyg1 — APPROVED** [review]

**Pull request merged**